core-bundle/contao/library/Contao/StringUtil.php

```diff
@@ -228,7 +228,7 @@ public static function decodeEntities($strString, $strQuoteStyle=ENT_QUOTES)
 	 */
 	public static function convertBasicEntities($strBuffer)
 	{
-		return str_replace(array('&amp;', '&lt;', '&gt;', '&nbsp;', '&shy;'), array('[&]', '[lt]', '[gt]', '[nbsp]', '[-]'), $strBuffer);
+		return str_replace(array('&amp;', '&lt;', '&gt;', '&nbsp;', '&shy;', '&#8203;'), array('[&]', '[lt]', '[gt]', '[nbsp]', '[-]', '[ ]'), $strBuffer);
```

Comment:
```suggestion
		return str_replace(array('&amp;', '&lt;', '&gt;', '&nbsp;', '&shy;', '&ZeroWidthSpace;'), array('[&]', '[lt]', '[gt]', '[nbsp]', '[-]', '[zwsp]'), $strBuffer);
```

I’m not sure if we should add this at all, but if so I think we should name it `[zwsp]`

Reply:
I agree `zwsp` would be less of a conflict and is the official entity name. I would prefer `wbr` though, as in the HTML standard: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr

Reply:
I'd only choose `[wbr]` if we actually replace it with `<wbr>`. If we replace it with `&zwsp` or the actual unicode character then I also prefer `[zbsp]`, to keep it also consistent with `[nbsp]`.

Reply:
Basic entities cannot be tags IMO so I’d vote for using `&ZeroWidthSpace;` and `[zwsp]`.